streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1604,7 +1604,9 @@ public KafkaFuture<Map<String, KafkaFuture<Uuid>>> producersClientInstanceIds(fi
         }
 
         if (processingMode.equals(StreamsConfigUtils.ProcessingMode.EXACTLY_ONCE_ALPHA)) {
-            throw new UnsupportedOperationException("not yet implemented");
+            log.warn("EOS v1 enabled: Producer client instance IDs are not collected." +
```

Comment:
I'd use ERROR, since this will only affect people who do want to use the new API and use EOSv1, so we should make this quite prominent.